test/sailci.test.js

```diff
@@ -0,0 +1,23 @@
+import test from 'ava';
+import sailci from '../lib/sailci';
+
+/* eslint-disable camelcase */
```

Comment:
This eslint-disable shouldn't be necessary here as there is no lower camel-case props.